esp/logging/loggingagent/cassandraloggingagent/cassandralogagent.cpp

```diff
@@ -206,14 +250,17 @@ void CCassandraLogAgent::executeUpdateLogStatement(StringBuffer& st)
     cassSession->disconnect();
 }
 
-unsigned CCassandraLogAgent::executeSimpleSelectStatement(const char* st, unsigned& resultValue)
+bool CCassandraLogAgent::executeSimpleSelectStatement(const char* st, unsigned& resultValue)
```

Comment:
the added logic changes the method's functionality slightly and we might want the name to reflect that. I think this method is now verifying target table not empty?